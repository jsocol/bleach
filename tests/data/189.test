<div id="122"><iframe sandbox="allow-same-origin allow-forms allow-scripts" src="http://example.org/"></iframe>//["'`-->]]>]</div>
