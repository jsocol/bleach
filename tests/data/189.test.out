<div id="122">//["'`--&gt;]]&gt;]</div>
